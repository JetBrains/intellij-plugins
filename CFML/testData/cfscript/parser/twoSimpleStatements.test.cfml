<cfscript>
a = 1;
b();
</cfscript>