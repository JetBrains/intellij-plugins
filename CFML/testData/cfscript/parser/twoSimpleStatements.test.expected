CfmlFile:twoSimpleStatements.cfml
  Tag
    PsiElement(OPENER)('<')
    PsiElement(CF_TAG_NAME)('cfscript')
    PsiElement(R_ANGLEBRACKET)('>')
    Assignment
      ReferenceExpression
        PsiElement(IDENTIFIER)('a')
      PsiElement(ASSIGN)('=')
      IntegerLiteral
        PsiElement(INTEGER)('1')
    PsiElement(SEMICOLON)(';')
    FunctionCallExpression
      ReferenceExpression
        PsiElement(IDENTIFIER)('b')
      ArgumentList
        PsiElement(L_BRACKET)('(')
        PsiElement(R_BRACKET)(')')
    PsiElement(SEMICOLON)(';')
    PsiElement(LSLASH_ANGLEBRACKET)('</')
    PsiElement(CF_TAG_NAME)('cfscript')
    PsiElement(CLOSER)('>')